exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how is the level of complexity of a given text assessed, and how can we make programs which do that? 
```

Comment:
Please wrap sentences consistently :)

If you're wrapping at column offsets, you may need to reformat paragraphs every time they change.
If you wrap by sentences, it tends to be less work, but less uniform width.